packages/app-bridge/src/react/useGuidelineActions.ts

```diff
@@ -503,7 +503,7 @@ export const useGuidelineActions = (appBridge: AppBridgeTheme) => {
 
     const moveDocumentPageBetweenDocuments = useCallback(
         async (id: number, sourceDocumentId: number, targetDocumentId: number) => {
```

Comment:
Can we not remove the `sourceDocumentId` here as well? I don't think it's needed.

```suggestion
        async (id: number, targetDocumentId: number) => {
```

Reply:
It is needed for the emitter, we want to update both target and source documents, and we don't know the source document otherwise